policy-controller/src/main.rs

```diff
@@ -239,16 +262,23 @@ impl std::str::FromStr for IpNets {
 async fn grpc(
     addr: SocketAddr,
     cluster_networks: Vec<IpNet>,
-    index: SharedIndex,
+    inbound_index: SharedIndex,
+    outbound_index: outbound_index::SharedIndex,
     drain: drain::Watch,
 ) -> Result<()> {
-    let discover = IndexDiscover::new(index);
-    let server = grpc::Server::new(discover, cluster_networks, drain.clone());
+    let inbound_discover = IndexDiscover::new(inbound_index);
+    let inbound_svc =
+        grpc::InboundPolicyServer::new(inbound_discover, cluster_networks, drain.clone()).svc();
+
+    let outbound_discover = OutboundDiscover::new(outbound_index);
```

Comment:
IndexDiscover vs OutboundDiscover -- inconsistent. How do we make this consistent?